NDB.Covid19/NDB.Covid19/Enums/NotificationsEnum.cs

```diff
@@ -42,15 +42,13 @@ public static NotificationViewModel Data(this NotificationsEnum notificationType
                     return new NotificationViewModel
                     {
                         Type = NotificationsEnum.ConsentNeeded,
-                        Title = "NOTIFICATION_UPDATE_HEADER".Translate(),
-                        Body = "NOTIFICATION_CONSENT_DESCRIPTION".Translate()
+                        Title = "NOTIFICATION_UPDATE_HEADER".Translate()
                     };
                 case NotificationsEnum.ReApproveConsents:
                     return new NotificationViewModel
                     {
                         Type = NotificationsEnum.ReApproveConsents,
-                        Title = "NOTIFICATION_CONSENT_HEADER".Translate(),
-                        Body = "NOTIFICATION_CONSENT_DESCRIPTION".Translate()
```

Comment:
Why you have removed body of the notification? we cannot have notification without it. however those 2 notifications are not relevant for release 1.

Reply:
updated